### ISSUE TITLE
Explain 'type area'

### ISSUE THREAD

**Issue by r12a** [w3c/clreq#56]
http://w3c.github.io/clreq/#type_area
http://w3c.github.io/clreq/zh/#type_area

the first para in 2.3 The Type Area says: "The type area, sometimes called the printing area, is designed in the following sequence."

There is no actual definition of what the type area is. I suggest we change the foregoing to something like: "The type area, sometimes called the printing area, is the rectangle in the middle of the page that contains the main body of the text. It is surrounded by space containing headers, footers, notes, etc. It is designed as follows:"

**Comment by realfish** [comment]
If I remember correctly, _The Elements of Typographic Style_ uses the term “textblock”, which could be well mapped to the concept of “版心”.

**Comment by r12a** [comment]
Page 145 of the 4th edition carries a margin note that says:
"The textblock is known in Chinese as yèxǐn (頁心), a useful phrase. Yè means 'page'; xīn means 'heart and mind'."

The usage of 'textblock' in Bringhurst, however, seems to match the usage of 版心 (type area) in clreq (and kihon-hanmen 基本版面 in jlreq).

My main point, however, is that we should add a definition of the term to the paragraph.

**Comment by bobbytung** [comment]
My opinion:
- Richard's definition is fine but 'type area' do not equal to 'printing area'.
- Why they use Kihon-hanmen in JLREQ? Could we use Banxin is same manner?

**Comment by ethantw** [comment]
If we use Banxin, we would have to deal with the same dilemma we met on the 漢字(Hanzi/Han characters) issue.

**Comment by realfish** [comment]
I think first, it seems necessary to clarify **the meaning and the usage** of “版心” in _CLReq_’s Chinese version.

Usually, “版心/頁心”, or the term “textblock” used by _ETS_, seems much simpler than “kihon-hanmen” in _JLReq_. It’s just focused on describing **the bounding box of the body text** rather than something more. If so, “textblock” seems good enough for the interpretation sake. (_ETS 4.0_ presents the form of textblock in § 8.4, which could be a nice reference.)

If “版心” (banxin) is supposed to be the term corresponding to the “kihon-hanmen”,  maybe it needs more interpretation to **redefine this term both in Chinese and English**.

---

I consider kihon-hanmen is one kind of basic grids.
- For Latin scripts, the basic grid, which is also known as the “baseline grid”, is defined by **the baselines of body text**. Its size and rhythm are directly defined by the line-height of the body text.
- For JP and CN, the basic grid is defined by a set of solid-size square boxes. **This solid-size square box, named “character frame” in _JLReq_, is the crucial element for this kind of grid.** The size and rhythm of this grid are defined by the body font size with the line-gap.

Kihon-hanmen (基本版面) is almost an alias of the Japanese basic grid mentioned above. In Adobe InDesign, it translates as “レイアウトグリッド” (layout grid) in JP version and “版面网格” in CN version. Here, the phrasing of “グリッド・网格” (grid) is critical.

All the _grids_ above emphasize a common point — the rhythm of typesetting is based on the dimensions of body text.

**Comment by ryukeikun** [comment]
Referring JLReq, there are 2 _separate_ terms:
- **hanmen (page content area)  版面  hanmen**    Actual printed area in a page excluding the margins. (note: Running heads and page numbers are not part of hanmen.) (JIS Z 8125)
- **kihon-hanmen    基本版面    kihon hanmen**  The default dimensions of the main area of a typeset page specified by text direction, number of columns, character size, number of characters in a line, number of lines in a column, inter-line spacing and inter-column spacing. (JIS X 4051)

and things are easy to confuse but
1. hanmen（版面）in Japanese, 版心 or 頁心 in Chinese, and in English, this term can be textblock in _ETS_, but we must also notice that in GB 9851.2-2008（印刷技术术语）, there is a definition as below, which perfectly matches the term _hanmen_ in JIS Z 8125:

>  3.3 版心 type page 印刷成品版面中的图文印刷区域（不含出血图像）

cf.

> 3.2 版面 type area 印刷成品幅面中，图文和空白部分的总和。

To make everything clear, let me repeat:
版面 (banmian) in Chinese is  紙面 (shimen) in Japanese
版面 (hanmen) in Japanese is 版心(banxīn) in Chinese

Basically, according to GB definition above the area of 版面 (banmian) is larger than 版心(banxīn).
1. kihon-hanmen 基本版面 in Japanese, maybe 基本排版体裁 in Chinese, and in English it should be something like “Basic style of type page”. According to the definition of JIS X 4051, “layout grid” may be _one of_ the approach of kihon-hanmen. 

As far as I know, Japanese Layout is much rely on “layout grid”, but not Modern Chinese. As a matter of fact, a lot of Modern Chinese layout can be processed with Western “Margins And Columns” mode in Adobe InDesign, while the approach and result that strictly follow the “layout grid” is 纵横对齐 (grid alignment).

**Comment by xfq** [comment]
During the teleconference, we decided to use Richard's proposed text, or the current definition in the [Glossary](https://w3c.github.io/clreq/#glossary).

@r12a what do you think?

**Comment by xfq** [comment]
The English text seems to be fixed already. I'll close this issue and submit a PR to add the Chinese text.
